src/index.js

```diff
@@ -96,9 +96,14 @@ var OAuth = function OAuth(clientId) {
 	this.refreshToken = null;
 	this.tokenType = null;
 	this.expiresIn = 0;
-	this.ignoreSslError = false;
 };
 
+
+/**
+ * For by passing self signed certificate. By default false
+ */
+OAuth.prototype.ignoreSslError = false;
```

Comment:
Look at line 99 that you removed. You can set it the same way as your implementation, we just define the property and it's default in the constructor rather than separately like this.